other/max_sum_sliding_window.py

```diff
@@ -0,0 +1,43 @@
+"""
+Given an array of integer elements and an integer 'k', we are required to find the
+maximum sum of 'k' consecutive elements in the array.
+
+Instead of using a nested for loop, in a Brute force approach we will use a technique
+called 'Window sliding technique' where the nested loops can be converted to a single
+loop to reduce time complexity.
+"""
+import sys
+from typing import List
+
+
+def max_sum(array: List[int], k: int) -> int:
+    """
+    Returns the maximum sum of k consecutive elements
+    >>> arr = [1, 4, 2, 10, 2, 3, 1, 0, 20]
+    >>> k = 4
+    >>> max_sum(arr, k)
+    24
+    >>> k = 10
+    >>> max_sum(arr,k)
+    Invalid input
+    -1
+    """
+    if len(array) < k or k < 0:
+        print("Invalid input")
+        return -1
+    max_sum = -sys.maxsize

```

Comment:
Can you tell me why is this useful?
How is an extremely low number be `max_size`?

Reply:
You can see its `-max_size()`

Reply:
If you want a negative infinity number then just use `float('-inf')`

Reply:
This way is more commonly used, and so easy to understand

Reply:
> You can see its -max_size()

What do you mean by this?